javalib/src/main/scala/java/lang/Float.scala

```diff
@@ -85,7 +86,7 @@ object Float {
 
   @inline def valueOf(s: String): Float = valueOf(parseFloat(s))
 
-  private[this] lazy val parseFloatRegExp = new js.RegExp(
+  private[this] lazy val parseFloatRegExpStr =
```

Comment:
Same thing here. Go back to
```suggestion
  private[this] lazy val parseFloatRegExp = RegExpImpl.impl.compile(
```
?